backend/experiment/rules/toontjehoger_4_absolute.py

```diff
@@ -184,3 +185,20 @@ def get_final_round(self, session):
         )
 
         return [*score, final, info]
+
+    def validate_playlist(self, playlist: Playlist):
+        errors = super().validate_playlist(playlist)
+
+        # Get group values from sections, ordered by group
+        groups = list(playlist.section_set.values_list('group', flat=True).order_by('group'))
```

Comment:
This will give you a query set as follows:
```python
<QuerySet ['1', '1', '1', '10', '10', '10', '11', '11', '11', ...]>
```

Not sure if this is your intention here? It seems as if the unit test would fail if you went for values above 9.

Reply:
Fixed